### ISSUE TITLE
Invalid category

### ISSUE THREAD

**Issue by spharish** [hackerkid/Wikifeedia#1]
File : homepage.js
Line : 147 
category_of_page = search_input, if the search input is not valid category, then the further lines cannot go to the category which does not exist. So a message should be displayed which showing it is not a valid category.

**Comment by SaptakS** [comment]
I agree with @spharish . If the category has no string or a string which is not a valid category, it should be checked. Maybe on searching without a string you should load featured_post category.

**Comment by sabahmax-dev** [comment]
Hi! I’ve opened a pull request (#25) to fix this. It adds a check so that if the search input isn’t a valid category, the page displays a message instead of trying to load a non-existent category.
